docs/evm/Build/wallets-and-configurations.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Wallets & Configurations
+sidebar_label: Wallets & Configurations
+sidebar_position: 4
+---
+
+# Wallets & Configurations
+
+This document shows how to integrate the FlowEVM Network programmatically with your Dapp via MetaMask.
+
+## Metamask
+
+Integrating additional networks into MetaMask can pose challenges for users who lack technical expertise and may lead to errors. Simplifying this process can greatly enhance user onboarding for your application. This guide demonstrates how to create a straightforward button within your frontend application to streamline the addition of the Berachain network to MetaMask.
+
+### EIP-3035 & MetaMask
+
+[EIP-3035](https://eips.ethereum.org/EIPS/eip-3085) is an Ethereum Improvement Proposal that defines an RPC method for adding Ethereum-compatible chains to wallet applications. Since March 2021 MetaMask has implemented that EIP as part of their MetaMask [Custom Networks API](https://consensys.io/blog/connect-users-to-layer-2-networks-with-the-metamask-custom-networks-api).
+
+### Network configuration
+
+To add the FlowEVM network to Metamask, we need to add the following network confgiuration:
+
+#### Previewnet Network
+
+```js
+export const FLOWEVM_PREVIEWNET_PARAMS = {
+    chainId: '0x2eb',
+    chainName: 'Flow',
+    rpcUrls: ['https://previewnet.evm.nodes.onflow.org'],
+    nativeCurrency: {
+        name: 'Flow',
+        symbol: 'FLOW',
+        decimals: 18,
+    },
+    blockExplorerUrls: ['https://previewnet.flowdiver.io'],
+}
```

Comment:
I have set the block explorer to `https://previewnet.flowdiver.io` - this doesn't exist now, is this where the block explorer URL for previewnet will eventually be though?

Reply:
@bjartek can you confirm?

Reply:
@bjartek confirmed to me async earlier today